bookkeeper-server/src/main/java/org/apache/bookkeeper/bookie/SyncThread.java

```diff
@@ -108,14 +103,13 @@ public void startCheckpoint(Checkpoint checkpoint) {
         });
     }
 
-    public Future<Void> requestFlush() {
+    public Future requestFlush() {
```

Comment:
nit: why did you remove the generic here?

Reply:
1) it is returning void/null.
2) its return value is not used anywhere.
3) MockExecutorController doesn't support controlSubmitCallable.

Reply:
> it is returning void/null.

then you might consider returning Future<?>

> MockExecutorController doesn't support controlSubmitCallable.

oh, we can add support for submit callables